cmd/titus/gitlab.go

```
@@ -185,6 +195,25 @@ func runGitLabScan(cmd *cobra.Command, args []string) error {
+		if gitlabJitter > 0 {
+			cloneEnum.Jitter = time.Duration(gitlabJitter * float64(time.Second))
+		}
+
+		// Estimate scan time and prompt for confirmation if delay/jitter is enabled
+		if cloneEnum.Jitter > 0 || cloneEnum.Delay > 0 {
+			estimate := cloneEnum.EstimateScanTime()
+			if estimate > 0 {
+				_, _ = fmt.Fprintf(cmd.ErrOrStderr(), "Estimated scan time: %s (with %d projects)\n", formatDuration(estimate), len(projects))
+				_, _ = fmt.Fprintf(cmd.ErrOrStderr(), "Continue? [Y/n] ")
+				var response string
+				fmt.Scanln(&response)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Scanln` is not checked

(errcheck)